locales/en/statuses.json

```diff
@@ -6,6 +6,8 @@
   "requested": "Move requested",
   "accepted": "Move accepted",
   "completed": "Move completed",
+  "booked": "Booked",
```

Comment:
Could we use the existing pattern of prefixing statues with `Move` please. So these would become:

- `Move booked`
- `Move in transit`